packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.scss

```diff
@@ -18,9 +18,9 @@
 }
 
 button.check {
-  height: 20px;
-  width: 20px;
-  padding: 0;
+  height: 100%;
+  width: 40px;
+  padding: 6px 8px;
```

Comment:
Please don't use padding like this ever for things that are supposed to be fixed. Why 6px? Why 8? I think you are making an assumption about the size of the contents of the button... As soon as a designer changes the size of the check, the button will.

Instead, you should center the content within.